app/models/area.rb

```diff
@@ -3,6 +3,7 @@ class Area
   field :name, type: String
   field :pincode, type: Integer
   field :city_id, type: String
+  field :user_id, type: String
```

Comment:
while creation of this module associations was not that much of clear to but. bt nw its updated